tests/standalone_tests/testing_env_configs/test_localnodes_addresses.json

```diff
@@ -1 +1 @@
-["172.17.0.1:60000", "172.17.0.1:60001", "172.17.0.1:60002","172.17.0.1:60003"]
+["172.17.0.1:60000","172.17.0.1:60003"]
```

Comment:
This is used from the cleanup tests that are currently skipped. 

Make a different file and use that.